README.md

```diff
@@ -1,40 +1,119 @@
 # shield-study-rappor
 
-## install
+## Introduction
+This addon is a SHIELD Study that collects the eTLD+1 of the user's homepage
+and applies [RAPPOR](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/42852.pdf),
+a [Differential Privacy](https://en.wikipedia.org/wiki/Differential_privacy) algorithm, to encode it.
+The result is sent in a custom ping.
+
+The purpose of this study is to answer some questions with unbiased data such as:
+
+- "Which top sites are users visiting?"
+- "Which sites using Flash does a user encounter?"
+- "Which sites does a user see heavy Jank on?" 
+
+## Addon Behaviour
+The addon collects the user's homepage from the `browser.startup.homepage` preference.
+Then, from this value, obtains the eTLD+1. Here, it can fail if the host is 
+an IP address or is empty when calling `Services.eTLD.getBaseDomain`. In such case, the
+study ends.
+Other possiblity is that the user's homepage is `about:home` or other `about:` page.
+In the case of `about:home`, this is the value we use. In the case of other `about:`
+page, the reported value is `about:pages`.
+
+Once the user's homepage eTLD+1 value is obtained, RAPPOR is executed with the 
```

Comment:
Since we mentioned this earlier, we don't need it here.

Reply:
Done